CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,64 @@
+* **We are kind**. We are warm, courteous, patient, respectful and encouraging. Disagreements, both social and technical, happen all the time and Cucumber is no exception. These differences are where we can learn the most — if everyone thought the same all the time life would be boring! We’re respectful of other people’s opinions, experience, their skills, their commitments and their efforts. We make an effort to be compassionate towards one another, to assume good intention of others, and to seek to understand first before seeking to be understood.
+* **We are tolerant**. A diverse group of people has a diverse set of boundaries about what is OK. Sometimes one of us may make a mistake, and do something that inadvertently causes offence. When we observe this happen, we try to give the offender feedback quickly, clearly and constructively. We try to offer them the opportunity to maintain their dignity in the situation, to learn from and rectify their mistake.
+* ** We do not tolerate unpleasant behaviour**. If someone repeatedly or deliberately acts in an offensive way, we will take decisive action to prevent any further harm.
+
+## What is not OK
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a guide to make all of our daily interactions more straightforward and enriching.
+
+* Aggressive behaviour, whether in words or actions, whether overt or passive, is not OK.
+* In other words: don’t be mean, rude, demeaning, condescending or demanding.
+* Don’t discriminate against anyone.
+* Don’t shame people for their mistakes or inexperience.
+* Don’t harass people, sexually or otherwise. If someone has asked you to stop doing whatever you're doing - or if you notice they don't like what you're doing - stop and apologize, even if it seems unreasonable to you. Then remember, they are not under any obligation to accept your apology or interact with you in the future. Harassment includes comments or questions related to personal attributes such as (but not limited to) age, body size, culture, ethnicity, gender expression, gender identity, level of experience, nationality, personal ability or disability, physical appearance, physical or mental difference, race, religion, set of skills, sexual orientation, socio-economic status, and subculture. Harassment also includes sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, inappropriate physical contact, and unwelcome sexual attention.
```

Comment:
In the current internet climate, I think it can't hurt to be redundant.  This is a policy for people, not computers.  Redundancy makes it more effective.